cruise-control/src/main/java/com/linkedin/kafka/cruisecontrol/monitor/task/LoadMonitorTaskRunner.java

```diff
@@ -189,7 +197,8 @@ private void loadSamples() {
       _samplingScheduler.submit(new SampleLoadingTask(_sampleStore,
                                                       _partitionMetricSampleAggregator,
                                                       _brokerMetricSampleAggregator,
-                                                      this));
+                                                      this,
+                                                      _executor));
```

Comment:
[`This can potentially be done on a separate issue`] Looks like this function (i.e. `loadSamples()`) is called only when `skipLoadingSamples` is `false` in `#start(boolean skipLoadingSamples)` -- I am curious if we can make this value configurable (currently `false` is hard-coded)?

Reply:
Will do in another patch.